providers/FeedProvider.tsx

```diff
@@ -91,6 +96,41 @@ const FeedProvider = ({ children }: { children: ReactNode }) => {
     [feed, filter],
   );
 
+  const playFeedId = async (feedIndex: number) => {
+    const embed = findValidEmbed(feed[feedIndex]);
+    const feedObj = feed[feedIndex];
+    const url = embed?.url;
+    if (url) {
+      const metadata = await fetchMetadata(url, feedObj);
+      if (metadata)
+        dispatch({
+          type: 'PLAY',
+          payload: { metadata, feedId: feedObj.id },
+        });
+    }
+  };
+
+  const handleNext = async () => {
+    const feedIndex = feed.findIndex((feedObj: SupabasePost) => feedObj.id === player.feedId);
+
+    if (feedIndex > -1) {
+      if (feedIndex + 1 < feed.length) {
+        playFeedId(feedIndex + 1);
+      }
+    } else if (feedIndex <= -1) {
+      playFeedId(0);
+    }
+  };
```

Comment:
the `else if` is redundant and overcomplicates code which could be simpler

```suggestion
  const handleNext = async () => {
    const feedIndex = feed.findIndex((feedObj: SupabasePost) => feedObj.id === player.feedId);

    if (feedIndex > -1) {
      if (feedIndex + 1 < feed.length) {
        playFeedId(feedIndex + 1);
      }
      return
    } 
      playFeedId(0);
  };
```

same logic with less code
- less is more
- simple is best
- small is beautiful